### PR TITLE
Reintroduce a proper way to compute avg_loss

### PR DESCRIPTION
**Pull request by jhauglid**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -183,10 +183,11 @@ def lr_drop(epoch):
             optimizer.step()
 
         loss = losses.mean()
```
Comment (rune-mazemap) [inline train.py]: `loss` should be renamed `mean_losses` or somesuch here.

Comment (jhauglid) [reply train.py]: Updated

**Review by rune-mazemap — APPROVED** [review]
